spec/menu_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'menu'
+
+describe Menu do
+
+  let(:biriyani){double :biriyani, title: "Biriyani", price: 5.00}
+
+  context 'when created' do 
+    it 'has a default set of dishes' do
+      expect(subject.dishes).to eq Menu::DEFAULT_MENU
+    end
+
+    xit 'displays itself'
+      # Don't know how to test for this
+  end
+
+  it 'can have dishes added to it' do
+    subject.add_dish(biriyani, 5.00)
+    expect(subject.dishes).to include biriyani
+  end
+
+  describe '#show' do 
+    it 'returns an ASCII table representation of the menu using the terminal-table gem' do 
+      # I don't know how to test for this
+      expect(subject).to respond_to :show
```

Comment:
Ahhh, OK. Check the terminal-table docs and construct your own variant example, then test that `show` outputs a table that looks like that.

You might want to extract your own variant example to a private method in this `describe` block.